test/decoders/test_video_decoder_ops.py

```diff
@@ -84,6 +84,21 @@ def test_seek_and_next(self, device):
         )
         frame_compare_function(frame_time6, reference_frame_time6.to(device))
 
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    def test_seek_to_negative_pts(self, device):
+        device = "cpu"
```

Comment:
Doesn't think override the parameterization of the test?

Reply:
Good catch. Yeah, forgot to remove it before pushing to my fork